static/js/src/contextual-menu.js

```diff
@@ -1,7 +1,5 @@
 function toggleMenu(element, show) {
-  const dropdown = document.querySelector(
-    element.getAttribute("aria-controls")
-  );
+  const dropdown = element.nextElementSibling;
```

Comment:
This much more fragile then what was there. Why not use `aria-controls`?

Reply:
This was accidentally left in and should have been reverted back to using `aria-controls` - will fix.